### PR TITLE
Load version label from airgap metadata if it's available

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: ffi/airgap.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func PullFromAirgap(socket, licenseData, airgapDir, downstream, outputFile, regi
 			Downstreams:         []string{downstream},
 			LocalPath:           releaseDir,
 			LicenseFile:         licenseFile,
+			AirgapMetaFile:      filepath.Join(airgapDir, "airgap.yaml"),
```
Comment (marccampbell) [inline ffi/airgap.go]: @divolgin thinking about this and changes in the future... should we just walk the filetree of the archive, matching GVK instead?

File: pkg/upstream/fetch.go [hunk 2]
```diff
@@ -43,7 +44,7 @@ func downloadUpstream(upstreamURI string, fetchOptions *FetchOptions) (*Upstream
 		return downloadHelm(u, fetchOptions.HelmRepoURI)
 	}
 	if u.Scheme == "replicated" {
-		return downloadReplicated(u, fetchOptions.LocalPath, fetchOptions.RootDir, fetchOptions.UseAppDir, fetchOptions.License, fetchOptions.ConfigValues, fetchOptions.CurrentCursor, fetchOptions.CurrentVersionLabel)
+		return downloadReplicated(u, fetchOptions.LocalPath, fetchOptions.RootDir, fetchOptions.UseAppDir, fetchOptions.License, fetchOptions.ConfigValues, fetchOptions.CurrentCursor, pickVersionLabel(fetchOptions))
```
Comment (marccampbell) [inline pkg/upstream/fetch.go]: I think we should update the tests on `downloadReplicated` (or create if necessary) to handle this new functionality. This seems like an area that's likely to break if we don't have a test around it.

Comment (divolgin) [reply pkg/upstream/fetch.go]: 👍

File: ffi/airgap.go (hunk 1 above)
Comment (divolgin) [reply ffi/airgap.go]: @marccampbell this file is never in a subdirectory.... I think if we list files in the root of the bundle that should be sufficient.

**Pull request merged**